stack-overflow.txt

```diff
@@ -0,0 +1,19 @@
+I fly like a butterfly, fluttering in the air.
+But when all seems lost, I find a way.
+From the help of others, I rise again.
+Stack Overflow, here at our rescue!
```

Comment:
Oops, the pronoun suddenly switches to third person. Oh well, what's life without mistakes